src/internal/packager2/pull.go

```diff
@@ -112,12 +122,14 @@ func pullOCI(ctx context.Context, src, tarDir, shasum string, filter filters.Com
 		src = fmt.Sprintf("%s@sha256:%s", src, shasum)
 	}
 	arch := config.GetArch()
-	remote, err := zoci.NewRemote(ctx, src, oci.PlatformForArch(arch))
+	platform := oci.PlatformForArch(arch)
+	remote, err := zoci.NewRemote(ctx, src, platform)
 	if err != nil {
 		return false, "", err
 	}
 	desc, err := remote.ResolveRoot(ctx)
 	if err != nil {
+		l.Error("unable to resolve oci descriptor", "os", platform.OS, "arch", platform.Architecture, "error", err)
```

Comment:
The average user probably won't know what an OCI descriptor is. maybe something like `failed to pull package for platform {OS}/{ARCH}. 

I also think the error below is misleading. ResolveRoot is trying to get the manifest not the index.json. IMO combining this line and the below line would be the way to go for now

Reply:
I also feel like our error here is misleading, could you clarify what you mean by combining the two lines though?

Reply:
Combining these two lines
```go
		l.Error("unable to resolve oci descriptor", "os", platform.OS, "arch", platform.Architecture, "error", err)
		return false, "", fmt.Errorf("could not fetch images index: %w", err)
```
maybe something like
```go
return false, "", fmt.Errorf("could not find package at %s with architecture %s: %w", err, src, platform.Architecture)
```
Leaving out the `platform.OS` might make sense as well since from a user perspective since it's always `MultiOS` for Zarf packages which feels like more of an implementation detail.

Reply:
That's fair, definitely agreed on OS not being actionable info for users. I'll go ahead and take out the error log and we'll rely on the err log near main. Fixed in latest